gatorgrouper/static/gatorgrouper/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)

```

Comment:
What is this license for?

Reply:
I believe that comes with the bootstrap or the scss that we had, I am getting rid of it now